scripts/build-all.sh

```diff
@@ -37,14 +37,7 @@ RETVAL=$?
 [ $RETVAL -ne 0 ] && echo "Failure in building of core" && exit -1
 
 echo "Core streamlet libraries built, tested and published to local"
-echo "Now starting build of installer"
 
-cd ../installer
```

Comment:
this should have been catch in CI but it didn't :-(
we should reinforce those scripts with something like `set -euxo pipefail`. ( ref. https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/ )